### DISCUSSION TITLE
Why is Electron support being dropped in v3?

### DISCUSSION THREAD

**Discussion by reed-lawrence** [ionic-team/capacitor#4247] [General]
I see that Electron support has been moved to capacitor-community/electron, and there is no more mention of it in the Capacitor docs. 

I am still using an older version of capacitor that supports Electron via the CLI, but I would like to upgrade eventually. Why was support dropped for it?

**Comment by jcesarmobile** [comment]
It's not dropped, it's just maintained by the community now

**Reply by reed-lawrence** [reply]
Correct, but community maintenance is certainly different than official support. I'm just curious why it will no longer be maintained by the capacitor team?

**Reply by jcesarmobile** [reply]
Electron was created by a community member to begin with. And he needed to make breaking changes on it, but since on capacitor all packages share versions, it would mean all packages would need the major version bump to 3.0, but the other packages weren’t ready yet, so it was decided to make it a community platform, because, as I said, most work on it was done by a community member, who is maintaining now the capacitor-community platform.

**Reply by reed-lawrence** [reply]
Appreciate the insight, thanks!

**Reply by ProfVenturi** [reply]
> Electron was created by a community member to begin with. And he needed to make breaking changes on it, but since on capacitor all packages share versions, it would mean all packages would need the major version bump to 3.0, but the other packages weren’t ready yet, so it was decided to make it a community platform, because, as I said, most work on it was done by a community member, who is maintaining now the capacitor-community platform.

Good morning, excuse me; I'm a teacher trying to illustrate to my students the use of Electron in Ionic Capacitor projects. Can you please comment on the following:
![image](https://github.com/ionic-team/capacitor/assets/150172025/eef9ff91-6200-4e4c-9248-1842f0c0041b)
I'm asking because I can't seem to be able to use the current capacitor version because of the following:
https://github.com/capacitor-community/electron/issues/284
I installed version 20 of NodeJS and it works, but then the Electron "to .exe" packagers don't work and I'm stuck.
Thanks for your explanation.

**Comment by francescodoffizi** [comment]
In capacitor/cli 3 it seems you _forgot_ to add in declarations.ts so no way to set-up capacitor-community/electron properly

electron?: {
        // Custom scheme for your app to be served on in the electron window.
        customUrlScheme?: string, 
        // Switch on/off a tray icon and menu, which is customizable in the app.
        trayIconAndMenuEnabled?: boolean,
        // Switch on/off whether or not a splashscreen will be used.
        splashScreenEnabled?: boolean,
        // Custom image name in the electron/assets folder to use as splash image (.gif included)
        splashScreenImageName?: string,
        // Switch on/off if the main window should be hidden until brought to the front by the tray menu, ect.
        hideMainWindowOnLaunch?: boolean,
        // Switch on/off whether or not to use deeplinking in your app.
        deepLinkingEnabled?: boolean,
        // Custom protocol to be used with deeplinking for your app.
        deepLinkingCustomProtocol?: string,
      };

**Reply by IT-MikeS** [reply]
TS will be mad yes, but the options will still function in the platform.
